mithril-common/src/sqlite/entity.rs

```diff
@@ -13,10 +13,6 @@ pub enum HydrationError {
     /// data are missing
     #[error("some data are missing: {0}")]
     MissingData(String),
```

Comment:
```HydrationError::MissingData``` seems not to be used. Is it worth keeping it?

Reply:
If it's not used, it's useless. I remove it.